tests/core/mempool/test_mempool_manager.py

```diff
@@ -534,3 +535,89 @@ async def test_get_items_not_in_filter() -> None:
     sb2_and_3_filter = PyBIP158([bytearray(sb2_name), bytearray(sb3_name)])
     result = mempool_manager.get_items_not_in_filter(sb2_and_3_filter)
     assert result == [sb1]
+
+
+def always(mm: MempoolManager, mi: MempoolItem) -> bool:
+    return True
+
+
+def never(mm: MempoolManager, mi: MempoolItem) -> bool:
+    return False
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize("test_for", ["max_cost", "max_fee", "normal_path"])
```

Comment:
Not needed, but you could make these an enum.

Reply:
Done.